esp/services/ws_packageprocess/ws_packageprocessService.cpp

```diff
@@ -31,6 +31,25 @@
 #include "exception_util.hpp"
 #include "TpWrapper.hpp"
 
+
+static StringBuffer & getRoxieDefaultPlane(StringBuffer & plane, const char * roxieName)
```

Comment:
NOTE: This function should be commoned up.  Tony has suggested a file it can probably live in - I will try moving it.

Reply:
now moved and extended to allow the plane to be configured